### ISSUE TITLE
access limit for side w/o data doesn't honor the `reverse` argument

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/gsplot#356]
`gs_3$side.3$reverse` is true, but the actual plot does not reflect this. Also, `xlim(gs_3)` has `side.3 = c(NA,NA)`

``` r
gs_3 <- gsplot() %>%
    points(1:10, 1:10) %>%
    axis(1, at = seq(0,10,by=0.1),labels=FALSE, tcl=0.15) %>%
    axis(3, reverse=TRUE)
gs_3

gs_4 <- gsplot() %>%
    points(1:10, 1:10) %>%
    axis(1, at = seq(0,10,by=0.1),labels=FALSE, tcl=0.15) %>%
    axis(3, reverse=TRUE)
gs_4
```

![image](https://cloud.githubusercontent.com/assets/13220910/16463787/f340fb3a-3dfc-11e6-972e-d3a31824179c.png)

**Comment by jordansread** [comment]
That's a pretty interesting case, and I am not sure what the expected behavior is. You don't have any data on axis 3, so reversing it is kind of misleading...not sure what to do in that case. warn?

**Comment by jordansread** [comment]
Also @lindsaycarr the case where you want the limits of a side that has no data to take on the properties of the opposite side with data, you use `xlim(gs_3, 1, set.undefined = TRUE)` the `set.undefined` arg

**Comment by lindsayplatt** [comment]
Add warning that no data is associated with side=3, so it inherits side=1 args. 

`xlim(gs)` should return 10,1 just like `xlim(gs,3)` does. Something weird with set.undefined is happeneing

**Comment by lindsayplatt** [comment]
Only gets to setting closest.side limits if there is a side specified because of `if (!is.null(side) && length(side)==1){` on https://github.com/USGS-R/gsplot/blob/master/R/access-gsplot.R#L124

**Comment by lindsayplatt** [comment]
``` r
gs <- gsplot() %>% 
    points(1:10,11:20, side=c(2,3))

#You get:
ylim(gs,4)
[1] 0 1
xlim(gs,1)
[1] 0 1

#But I would expect them to match the limits of the closest side:
ylim(gs,4)
[1] 11 20
xlim(gs,1)
[1] 1 10
```

**Comment by lindsayplatt** [comment]
@jiwalker-usgs and I decided we should actually expect NAs returned for non-existent sides

**Comment by lindsayplatt** [comment]
Fixed with #372
